uaclient/security_status.py

```diff
@@ -132,6 +136,42 @@ def get_ua_info(cfg: UAConfig) -> Dict[str, Any]:
     return ua_info
 
 
+def get_livepatch_fixed_cves() -> List[Dict[str, str]]:
+    try:
+        out, _err = subp([LIVEPATCH_CMD, "status", "--format", "json"])
+    except ProcessExecutionError:
+        return []
+
+    try:
+        livepatch_output = json.loads(out)
+    except json.JSONDecodeError:
+        msg = "Could not parse Livepatch Status JSON: {}".format(out)
+        logging.debug(msg)
+        return []
+
+    status_list = livepatch_output.get("Status")
+    if status_list:
+        kernel_info = get_kernel_info()
```

Comment:
`get_kernel_info` can also fail, although it should be rare, I think we should still handle it somehow.
We could try to run `uname -r` directly if it fails or just fail loudly if we have more than on dict on the status list

Reply:
Well the function calls `uname` as part of the execution through the Python code. 
I´d say if we get to the way weird scenario of not being able to parse the the Kernel information, logging it and returning empty list should be enough for this functionality (as there would be more to care about...)
Would that be enough?

Reply:
Yeah, I think that also works

Reply:
Well it seems (per MM conversation in ~livepatch) that the kernel check is useless anyway - I just removed it